awx/main/models/inventory.py

```diff
@@ -2616,6 +2616,9 @@ def build_script_private_data(self, inventory_update, private_data_dir):
         group_patterns = '[]'
         group_prefix = 'foreman_'
         want_hostcollections = 'False'
+        want_ansible_ssh_host = 'False'
+        rich_params = 'False'
+        want_facts = 'True'
```

Comment:
👍 to want_facts, because there is a clear corresponding parameter in the plugin

https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/inventory/foreman.py#L54

(have not really groked the details of what it does though)

Reply:
Note that the tower docs will need to be updated (for Tower 3.7) to reflect `want_facts` being (1) enabled by default and (2) a supported option.

https://docs.ansible.com/ansible-tower/latest/html/userguide/inventories.html#red-hat-satellite-6

Specifically, this call out:
![image](https://user-images.githubusercontent.com/4440360/72191297-9d0a8c00-33b6-11ea-8063-543066fb19ae.png)


cc @tvo318